peaseq-case.wdl

```diff
@@ -1,453 +1,576 @@
+                    else sub(basename(eachfastq), "_R?[12]_....f.*q.gz|_R?[12].f.*q.gz", ""
+                    ) + "/SpikeIn/FastQC",
             }
         }
         scatter (fastqpair in original_sample_fastqfiles) {
-            call util.basicfastqstats as spikein_indv_R1_bfs {
-                input :
-                    fastqfile=fastqpair.left,
-                    default_location=if multi_fastqpair then 'SAMPLE/' + sub(basename(fastqpair.left),'_R?[12]_....f.*q.gz|_R?[12].f.*q.gz','') + '/SpikeIn/SummaryStats' else if defined(results_name) then results_name + '/SpikeIn/SummaryStats' else sub(basename(fastqpair.left),'_R?[12]_....f.*q.gz|_R?[12].f.*q.gz','') + '/SpikeIn/SummaryStats'
+            call util.basicfastqstats as spikein_indv_R1_bfs { input:
+                fastqfile = fastqpair.left,
+                default_location = if multi_fastqpair then "SAMPLE/" + sub(basename(
+                    fastqpair.left), "_R?[12]_....f.*q.gz|_R?[12].f.*q.gz", "") + "/SpikeIn/SummaryStats"
+                    else if defined(results_name) then "~{results_name + '/SpikeIn/SummaryStats'}"
+                    else sub(basename(fastqpair.left), "_R?[12]_....f.*q.gz|_R?[12].f.*q.gz",
+                    "") + "/SpikeIn/SummaryStats",
             }
-            call bowtie.spikein_PE as spikein_indv_map {
-                input :
-                    fastqfile=fastqpair.left,
-                    fastqfile_R2=fastqpair.right,
-                    metricsfile=spikein_indv_R1_bfs.metrics_out,
-                    insert_size=insertsize,
-                    strandedness=strandedness,
-                    index_files=actual_spikein_bowtie_index,
-                    default_location=if multi_fastqpair then 'SAMPLE/' + sub(basename(fastqpair.left),'_R?[12]_....f.*q.gz|_R?[12].f.*q.gz','') + '/SpikeIn/SummaryStats' else if defined(results_name) then results_name + '/SpikeIn/SummaryStats' else sub(basename(fastqpair.left),'_R?[12]_....f.*q.gz|_R?[12].f.*q.gz','') + '/SpikeIn/SummaryStats'
+            call bowtie.spikein_PE as spikein_indv_map { input:
+                fastqfile = fastqpair.left,
+                fastqfile_R2 = fastqpair.right,
+                metricsfile = spikein_indv_R1_bfs.metrics_out,
+                insert_size = insertsize,
+                strandedness = strandedness,
+                index_files = actual_spikein_bowtie_index,
             }
         }
 
         if (length(original_sample_R1) > 1) {
-            call peaseq_util.sortfiles as spikein_R1_sorted { input: fastqfiles=spikein_indv_map.unaligned_1 }
-            call peaseq_util.sortfiles as spikein_R2_sorted { input: fastqfiles=spikein_indv_map.unaligned_2 }
+            call peaseq_util.sortfiles as spikein_R1_sorted { input: fastqfiles = spikein_indv_map.unaligned_1
+                }
+            call peaseq_util.sortfiles as spikein_R2_sorted { input: fastqfiles = spikein_indv_map.unaligned_2
+                }
```

Comment:
```suggestion
            call peaseq_util.sortfiles as spikein_R1_sorted { input:
                fastqfiles = spikein_indv_map.unaligned_1
            }
            call peaseq_util.sortfiles as spikein_R2_sorted { input:
                fastqfiles = spikein_indv_map.unaligned_2
            }
```
I think the current logic for "if a call has a single input, try and put it on a single line" is too faulty. I think it would be better to just always render calls as spanning multiple lines